docs/enhancements/day2-master.md

```diff
@@ -0,0 +1,45 @@
+---
+title: day2-master
+authors:
+- "@oamizur"
+  creation-date: 2022-07-10
+  last-updated: 2022-07-10
+---
+
+# Install day2 master
+
+## Summary
+
+Sometimes there is a need to install a day2 master to an existing cluster.  When using
+assisted installer, this flow is not supported.
+
+## Motivation
+
+There are several reasons there would be a need to install day master including:
+* An existing master node is malfunctioning
+* An existing master node does not meet new hardware requirements
+
+### Goals
+* Support installing either in SaaS or ACM
+* Support master node removal
+
+## Proposal
+
+There are several challenges in order to support day2 master installation:
+* Remove all traces from previous node that needs to be removed. These include: etcd, and that the node is completely down
```

Comment:
I thought we agreed that this would be a non-goal, at least for now?  The user should do this procedure along with etcd backup and whatever else is necessary.  We should just add the node and approve CSR, similar to workers.

Reply:
Yes